src/app/components/hero/index.tsx

```diff
@@ -0,0 +1,70 @@
+import * as React from 'react';
+import { Stack, Typography, Container, Divider } from '@mui/material';
+
+interface HeroProps {
+  imageUrl: string;
+  title: string;
+  subtitle?: string;
+  children?: React.ReactNode;
+}
+
+const Hero: React.FC<HeroProps> = ({ imageUrl, title, subtitle, children }) => {
+  return (
+    <Stack
+      sx={{
+        height: '500px',
+        backgroundImage: `url(${imageUrl})`,
+        backgroundSize: 'cover',
+        backgroundPosition: 'center',
+        position: 'relative',
+        color: 'white',
+      }}
+      // Using Stack properties to control layout
+      alignItems="center"
+      justifyContent="center"
+    >
+      {/* Overlay */}
+      <Stack
+        sx={{
+          position: 'absolute',
```

Comment:
🤔 seems a little bit odd... I'm not sure why the absolute positioning is being used here. Are you just wanting to center this element?

Reply:
Also seems odd to have a `Stack` without any child elements.. the whole point of Stack is to position child elements.

Reply:
Resolved by removing Box Component and adding        
 backgroundImage: `linear-gradient(rgba(0, 0, 0, 0.5), rgba(0, 0, 0, 0.5)), url(${imageUrl})`, to the the sx prop within Stack.
This provides the overlay effect.